pkg/resources/kafka/kafka.go

```diff
@@ -238,11 +238,23 @@ func (r *Reconciler) Reconcile(log logr.Logger) error {
 		}
 	}
 
-	// Handle PDB
+	// Handle PDB for brokers
 	if r.KafkaCluster.Spec.DisruptionBudget.Create {
-		o, err := r.podDisruptionBudget(log)
+		o, err := r.podDisruptionBudgetBrokers(log)
 		if err != nil {
-			return errors.WrapIfWithDetails(err, "failed to compute podDisruptionBudget")
+			return errors.WrapIfWithDetails(err, "failed to compute podDisruptionBudget for brokersr.KafkaCluster.Spec.KRaftMode ")
```

Comment:
Slight typo or did you mean to return the r.kafkaCluster.Spec.KRaftMode here as well

Reply:
It was a mistake in pasting, but the message should have been "failed to compute podDisruptionBudget for brokers"